lips/lip-20.md

```diff
@@ -0,0 +1,426 @@
+provided at the time of registration:
+
+| Name                          | Type                                   | Mutable      | Description                                                                                                             |
+| ----------------------------- | -------------------------------------- | ------------ | ----------------------------------------------------------------------------------------------------------------------- |
+| **to_lbr_exchange_rate**      | FixedPoint32                           | true         | The _rough_ exchange rate from `C` to LBR. Used only for dual attestation threshold and mempool gas price normalization   |
+| **is_synthetic**              | bool                                   | false        | Whether `C` is a synthetic currency or not (if false, then it is an SCS currency)                                       |
+| **scaling_factor**            | u64                                    | false        | The scaling factor that a coin's value in `C` should be multiplied by to arrive at the off-chain "real world" value     |
+| **fractional_part**           | u64                                    | false        | The smallest fractional part (number of decimal places) to be used in the human-readable representation of the currency |
+| **currency_code**             | vector\<u8\>                             | false        | The currency code (e.g., "LBR") for `C`                                                                                 |
+| total_value                   | u128                                   | true         | The total value of all currency in circulation on-chain, initialized to zero                                            |
+| preburn_value                 | u64                                    | true         | The total value of all coins in this currency waiting to be burned                                                      |
+| can_mint                      | bool                                   | true         | Whether more value in this currency can be added to circulation                                                         |
+
+All of the metadata about a specific currency `C` is held as part of the
+[`CurrencyInfo<C>`](https://github.com/libra/libra/blob/master/language/stdlib/modules/doc/Libra.md#resource-currencyinfo)
+resource that is uniquely published for each registered currency type `C` under the [Libra Root](https://github.com/libra/lip/blob/master/lips/lip-2.md#roles) (see also the implementation of
+[`Libra::register_SCS_currency`](https://github.com/libra/libra/blob/master/language/stdlib/modules/doc/Libra.md#function-register_scs_currency)
+and
+[`Libra::register_currency`](https://github.com/libra/libra/blob/master/language/stdlib/modules/doc/Libra.md#function-register_currency))
+account address. We now describe each of these data and their purpose.
+
+### `to_lbr_exchange_rate`
```

Comment:
I went for a table here since the `CurrencyInfo` struct contains a number of other fields related to events that I don't want to go into at this point in the document. I'm open to changing this to the full resource definition, but I do have a preference for the table though because of this. But, like I said, I'm open to changing it if you still think it should be the resource definition after this :)